modules/ossm-enabling-sidecar-injection.adoc

```diff
@@ -0,0 +1,275 @@
+// Module included in the following assemblies:
+// install/ossm-sidecar-injection
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-enabling-sidecar-injection_{context}"]
+= Enabling sidecar injection
```

Comment:
Why repeating what is described in the chapter above?
Also description in this chapter is not correct, it's mixing labels which can only be used on namespace with labels for pods. Also it's mixing labels used for default revision and labels used for non default revision.

Reply:
@FilipB I understand how this might cause confusion. I have removed the intro and added a normal one liner for the procedures. Hope it helps. Any information taken here was referenced from: https://github.com/openshift-service-mesh/sail-operator/blob/main/docs/ossm/injection/README.md

Please give your lgtm if it is okay now.